examples/UKCore-Condition-Extension-CodingSCTDescId-SN-MoleOfSkin-Ex.xml

```diff
@@ -0,0 +1,34 @@
+<Condition xmlns="http://hl7.org/fhir">
+	<id value="UKCore-Condition-Extension-CodingSCTDescId-SN-MoleOfSkin-Ex"/>
+	<!-- **************Snippet start************** -->
+	<code>
+		<coding>
+			<system value="http://read.info/ctv3"/>
+			<code value="X78Uv"/>
```

Comment:
X78uv is the CTV3 code for 255269000 | Late inspiration (observable entity)

XUYz6 is probably the code we want